chia/wallet/wallet_node.py

```diff
@@ -804,12 +831,16 @@ async def new_peak_wallet(self, new_peak: wallet_protocol.NewPeakWallet, peer: W
                     assert weight_proof is not None
                     old_proof = self.wallet_state_manager.blockchain.synced_weight_proof
                     if syncing:
-                        fork_point: int = max(0, current_height - 32)
+                        # This usually happens the first time we start up the wallet. We roll back slightly to be
+                        # safe, but we don't want to rollback too much (hence 32)
```

Comment:
You changed `-32` to `-16`, but the new comment says 32.

Reply:
Thank you i missed that